specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/examples/Compute/updateCustomServices.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "computeName": "compute123",
+    "api-version": "2023-04-01",
+    "customServices": [
+      {
+        "endpoints": [
+          {
+            "name": "lab",
+            "target": 8888,
+            "published": 8888,
+            "protocol": "http"
+          }
+        ],
+        "image": {
+          "type": "docker",
+          "reference": "docker.io/jupyter/minimal-notebook:latest"
+        },
+        "name": "jupyter"
+      },
+      {
+        "endpoints": [
+          {
+            "name": "connect",
+            "target": 8787,
+            "published": 8787,
+            "protocol": "http"
+          }
+        ],
+        "image": {
+          "type": "docker",
+          "reference": "docker.io/rocker/rstudio:latest"
+        },
+        "name": "rocker",
+        "title": "Rocker R ML IDE",
+        "description": "An open source R IDE."
```

Comment:
I don't see title and description properties defined in customService type.

Reply:
Those can go in additionalProperties. They are more like metadata for the users. Not compulsorily needed.

Reply:
On the second thought, removed them from examples.

Reply:
Thank you.